### PR TITLE
Allow passing through of crossorigin attribute

### PR DESCRIPTION
**Pull request by rubenstolk**

### CODE REVIEW

File: iron-image.html [hunk 1]
```diff
@@ -118,7 +118,7 @@
       hidden$="[[_computeImgDivHidden(sizing)]]"
       aria-hidden$="[[_computeImgDivARIAHidden(alt)]]"
       aria-label$="[[_computeImgDivARIALabel(alt, src)]]"></div>
-    <img id="img" alt$="[[alt]]" hidden$="[[_computeImgHidden(sizing)]]">
+    <img id="img" alt$="[[alt]]" hidden$="[[_computeImgHidden(sizing)]]" crossOrigin$="[[crossOrigin]]">
```
Comment (bicknellr) [inline iron-image.html]: I think the '`O`' in the attribute needs to be lowercase.

Comment (rubenstolk) [inline iron-image.html]: Hi @bicknellr thanks, indeed, amended the commit.

Comment (bicknellr) [inline iron-image.html]: We should make the property name on iron-image one word and lowercase to match.

Comment (rubenstolk) [inline iron-image.html]: OK did that!

**Comment by stramel** [comment]
@rubenstolk can you rebase this?

**Comment by rubenstolk** [comment]
Hey @stramel I've done that... but honestly not sure if the 7 months are worth the wait...

**Comment by stramel** [comment]
Yeah I hear ya @rubenstolk but this feels useful. Hopefully we can get it in. 

Ping @bicknellr @notwaldorf @e111077

**Comment by bicknellr** [comment]
This looks fine; sorry about the huge delay!

**Pull request merged**